Sources/Network/URLSessionNetworkStack.swift

```diff
@@ -134,9 +134,14 @@ public extension Network {
                     return
                 }
 
-                guard let httpResponse = response as? HTTPURLResponse else {
+
+
+                guard
+                    let urlResponse = response,
```

Comment:
if we now do this then most of the errors no longer require an `URLResponse?`, but a `URLResponse`, right?

Reply:
True, only the `.badResponse` needs an optional response 😃

Reply:
Actually, only `.http` and `.noData` can have non-optional responses.
And the `.authenticator` error does not feature a response associated value anymore.